extension.js

```diff
@@ -22,6 +22,7 @@ const Extension = imports.misc.extensionUtils.getCurrentExtension();
 const MenuItems = Extension.imports.menu_items;
 const Promise = Extension.imports.promise.Promise;
```

Comment:
Well, we're still doing `new Promise.all(requests).then(` a few lines later, which remained unchanged.